node/network/bridge/src/lib.rs

```diff
@@ -211,7 +211,7 @@ impl<N, AD> NetworkBridge<N, AD> {
 
 impl<Net, AD, Context> Subsystem<Context> for NetworkBridge<Net, AD>
 	where
-		Net: Network + validator_discovery::Network,
+		Net: Network + validator_discovery::Network + Sync,
```

Comment:
Not sure how to avoid `Sync` bound (requested by rustc after https://github.com/paritytech/substrate/pull/7247#discussion_r502263451), but it seems to compile with it.

Reply:
I think you don't need it if you do for example `let fut = sc_network::NetworkService::add_to_priority_group(&**self, group_id, multiaddresses); fut.await`, rather than combining them into one.

Reply:
Resolved by requiring `&mut self` instead, thanks!